addons/website_blog/static/src/js/website_blog.editor.js

```diff
@@ -186,8 +194,8 @@ options.registry.blog_cover = options.Class.extend({
         editor.on('save', this, function (image) {
             var src = image.src;
             this.$image.css('background-image', src ? ('url(' + src + ')') : '');
-            if (!this.$target.hasClass('cover')) {
-                var $opt = this.$el.find('[data-select-class]').first();
+            if (!this.$target.hasClass('o_wblog_has_cover')) {
```

Comment:
If you break compatibility, could you add the info in the task's pad so that we do not forget something during migration...?

Reply:
I added this in the task's pad, do you think is ok?

> Correctly prefix classes. eg 'cover' -> 'o_wblog_has_cover'

Reply:
Sure, I'll check once the task is merged ;)